storm-core/src/jvm/org/apache/storm/utils/IPredicate.java

```diff
@@ -15,20 +15,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 package org.apache.storm.utils;
 
-import org.apache.curator.framework.CuratorFramework;
-import org.apache.curator.framework.CuratorFrameworkFactory;
-import org.apache.curator.retry.ExponentialBackoffRetry;
-import java.util.Map;
-
-public class TestUtils extends Utils {
-
-    public static void testSetupBuilder(CuratorFrameworkFactory.Builder
-            builder, String zkStr, Map conf, ZookeeperAuthInfo auth)
-    {
-        setupBuilder(builder, zkStr, conf, auth);
-    }
-
+public interface IPredicate {
+    Boolean test (Object obj);
```

Comment:
Why return a big B Boolean and not a boolean?  When is null something we want to support in a predicate, and what does it even mean to return null?  Please either add javadocs or change it.